packages/kubernetes-1.21/kubernetes-1.21.spec

```
@@ -35,6 +35,9 @@ Source7: kubelet-bootstrap-kubeconfig
 Source8: kubernetes-tmpfiles.conf
 Source9: kubelet-sysctl.conf
 Source10: prepare-var-lib-kubelet.service
+Source11: kubernetes-tls-crt
+Source12: kubernetes-tls-private-key
```

In line with the output filename suggestions:
```suggestion
Source11: kubelet-server-crt
Source12: kubelet-server-key
```